addons/barcodes/__manifest__.py

```diff
@@ -15,7 +15,10 @@
         'web.assets_backend': [
             'barcodes/static/src/**/*',
         ],
-        'web.tests_assets': ['barcodes/static/tests/helpers.js'],
+        'web.tests_assets': [
+            'barcodes/static/tests/helpers.js',
+            'barcodes/static/tests/barcode_scan_tests.js'
```

Comment:
this bundle is for helpers, tests should be in the qunit_suite_tests bundle

Reply:
Hello, @aab-odoo 
I made the changes, please check this out.
Thank you!